src/XenCert/StorageHandlerUtil.py

```diff
@@ -935,9 +935,9 @@ def verify_data_on_vdi(session, vbd_ref, start_sec, end_sec):
 
         xencert_print('about to read from device: %s' % device)
 
-        expect = BUF_PATTERN
+        expect = BUF_PATTERN.encode()
```

Comment:
Just out of interest, what's the default encoding method (UTF-8 or ASCII)?

Reply:
UTF-8

Reply:
you should specify the encoding you want as otherwise it depends on the current execution environment.